### ISSUE TITLE
Monitor activity SSAS on Prem

### ISSUE THREAD

**Issue by HansPeetersconsulting** [DaxStudio/DaxStudio#924]
DAX Studio v3.0.0.725
Connection: AnalysisServices - 15.0.35.22

## Summary of Issue
I use the the functionality of "All queries" to monitor the activity of my on prem ssas cubes (SQL 2019).
When the cubes are accessed through PowerBI via a live connection everything works as expected, the same as the previous version of dax studio.
With the 3.0 version I dont see any Users for on prem access to the cubes.

## Steps to Reproduce
1. Start excel or powerbi desktop
2. Access an on prem cube
3. Now the query is visible in dax studio, but the type is Dax (was in the previous version MDX) and no user is visible

**Comment by marcosqlbi** [comment]
I can reproduce the issue, but it's something weird happening.
The user name appears if you open a SQL Profiler session and keep it running.
The user name returned in the profiler event is null if DAX Studio All Queries is the only active tracing.
@dgosbell do you think it could be something we do for the "View As" feature (that I'm not using)?
@HansPeetersconsulting a temporary workaround is to open a SQL Profiler session (you can use the SQL Profiler icon in DAX Studio) so you can see the user name.

**Comment by dgosbell** [comment]
I think this is a recent bug I introduced. I was having an issue with queries timing out, but I could not see the query text since All Queries was only capturing the QueryEnd event and this is not fired if the query times out. So I've modified the All Queries trace so that it initially captures the QueryBegin event, then when the query ends it updates the duration. But I was accidentally not capturing the username as part of the QueryBegin event. I've checked in a fix which will be part of the next release.

I'm not sure what you mean about the type previously being MDX - Power BI only generates DAX queries. But you might see MDX ones if you have other tools like Excel issuing queries.
